service/postgres/backup.go

```diff
@@ -49,6 +49,26 @@ func Backup(ctx context.Context, s3 *s3.Client, service config.Service, filename
 	}
 	command = append(command, "-c")
 	command = append(command, "--no-password")
+
+	// ssl/tls
+	if len(service.Binding.SSL.ClientCertPath) > 0 {
+		command = append(command, "sslcert="+service.Binding.SSL.ClientCertPath)
+	}
+
+	if len(service.Binding.SSL.ClientKeyPath) > 0 {
+		command = append(command, "sslkey="+service.Binding.SSL.ClientKeyPath)
+	}
+
+	if len(service.Binding.SSL.CACertPath) > 0 {
+		command = append(command, "sslrootcert="+service.Binding.SSL.CACertPath)
+	}
+
+	if service.Binding.SSL.VerifyServerCert {
+		command = append(command, "sslmode=verify-ca")
+	} else {
+		command = append(command, "sslmode=require")
```

Comment:
This command parameter should not be added by default, at least not unless there's also a check for `len()` of certs/keys. The default behaviour is expected to be entirely without SSL. If you check only for `VerifyServerCert`, then it will always add *"sslmode"* parameter and thus fail if there's no SSL at all. See failed CI build/test: https://app.circleci.com/pipelines/github/swisscom/backman/418/workflows/c170c089-8778-4d93-b5ff-f5c4588b1156/jobs/394?invite=true#step-109-51